### ISSUE TITLE
Try out Larynx for TTS

### ISSUE THREAD

**Issue by JarvyJ** [JarvyJ/HomeIntent#159]
This might only work on a Pi 4 (or maybe a 64bit pi 3), but I think Larynx might be a really good contender for voice options.

**Comment by oerkel47** [comment]
The only TTS I really liked was MaryTTS. It worked on a Pi4b but definitely took more time than PicoTTS.

**Comment by JarvyJ** [comment]
I could try out [OpenTTS](https://github.com/synesthesiam/opentts) and then select reasonable defaults based on language, architecture, speed, and auto-configure via the API. But that might be a project for another month.

**Comment by oerkel47** [comment]
I agree, it's just too dependent on the language. Probably should stick with what sounds reasonable in English as default. I think most users should be capable of changing things inside rhasspy. Related to that: I am not a huge fan of rhasspy settings being overwritten by default.

**Comment by JarvyJ** [comment]
Yeah, that's a tough one. On one hand, I want folks to have a nice install experience without having to configure things in multiple places, and on the other I want power users to be able to extend features as necessary. Unfortunately merging settings that are changed in two systems gets a little complicated...

Actually that gives me an idea on how to maybe use pydantic and avoid editing in two spots. I'll create an issue to investigate at some point. The short of it is that the HI defaults would be "known" (and are not overridable), and if anything has changed in Rhasspy to _not_ set it. I think it could work!

**Comment by oerkel47** [comment]
Sounds good. Or if that's too complicated maybe define some settings that only get set once or if undefined. And then only force the ones that are set via the HI ui/config.
